src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import "./App.css";
```

Comment:
Looks like you stripped ShadCN 99% of the way
would strip final 1% and remove dependency